storage/lib/storage.ex

```diff
@@ -0,0 +1,31 @@
+defmodule GoogleApi.Storage.Samples do
+  require Tesla
+
+  @moduledoc """
+  Documentation for GoogleApi.Storage.Samples.
+  """
+
+  @doc """
+  List storage buckets for a project.
+
+  ## Examples
+
+      iex> GoogleApi.Storage.Samples.list_buckets("your_project_id")
+      your_project_bucket1
+      your_project_bucket2
+
+  """
+  def list_buckets(project_id) do
+    {:ok, token} = Goth.Token.for_scope("https://www.googleapis.com/auth/cloud-platform")
+    conn = GoogleApi.Storage.V1.Connection.new(token.token)
+    {:ok, response} = GoogleApi.Storage.V1.Api.Buckets.storage_buckets_list(conn, project_id)
+    print_buckets(response.items)
```

Comment:
More idiomatic to use Enum.each

Reply:
I believe both using recursion and using `Enum.each` are considered idiomatic.

Reply:
Sure, you can write it with recursion, but this isn't doing anything special in the recursion. I'd probably write this as:

```elixir
Enum.each(response.items, &IO.puts(&1.id))
```

Reply:
Both are technically considered idiomatic I suppose (enumerations vs recursion, meh...), but `Enum.each` will be *much* more commonly used for such a case because it's more succinct and the intent is clear. I'd definitely recommend it for sample code.